WebContent/files/dams42json.xsl

```diff
@@ -168,14 +168,28 @@
 
     <xsl:template name="subject">
         <xsl:param name="columnName"/>
+        <xsl:variable name="hasExactExternalAuthority" select="mads:hasExactExternalAuthority/@rdf:resource" />
+        <xsl:variable name="fastHeadings">
+          <xsl:choose>
+            <xsl:when test="starts-with($hasExactExternalAuthority, 'http://id.worldcat.org/fast/')"> FAST</xsl:when>
```

Comment:
Are fast URI's always `http`? Are they ever `https`?

Reply:
@mcritchlow Good question. I am not sure whether it will be the https in the future or not but at this time it should be all http since https doesn't seem to be working. For example http://id.worldcat.org/fast/813106. But it won't harm to add the https case though. Would you like me to add it in?

Reply:
@lsitu - I'm not sure honestly. If they're always going to come in as `http` then it's not worth worrying about. @GregReser do you have any insight here?

Reply:
Not sure either.  Can you just test for "id.worldcat.org/fast" ?

Reply:
Sure. I'll change it to `id.worldcat.org/fast/` then.

Reply:
@mcritchlow I've added commit https://github.com/ucsdlib/damsmanager/pull/242/commits/9ceb9f0e211b53f6b116cd5fbfbdc10c9dcc69c0 to use the pattern `id.worldcat.org/fast/` for FAST subject headings instead.